app/src/main/java/org/mozilla/rocket/content/travel/ui/TravelMapper.kt

```diff
@@ -102,18 +104,21 @@ object TravelMapper {
                 video.linkUrl
             )
 
-    fun toHotelUiModel(hotel: Hotel): HotelUiModel =
-            HotelUiModel(
+    fun toHotelUiModel(hotel: BcHotelApiItem): HotelUiModel {
+
+        return HotelUiModel(
```

Comment:
You could simplify to `: HotelUiModel = HotelUiModel(...` like above `toVideoUiModel()`.